docs/getting_started.md

```diff
@@ -57,11 +57,11 @@ VAGRANTFILE_API_VERSION = "2"
 By default, port 8080 will be forwarded to port 80, and port 2121 will be forwarded to port 21 (for FTP),
 and 4096 MB of memory will be attributed to the VM.
 Enter the playbook directory `cd ansible-artimed` and type `vagrant up` to download a VM image and run the `galaxy.yml` playbook.
-This will take a while. Once finished, you should find a running Galaxy Instance on http://localhost:8080 .
+This will take a while. Once finished, you should find a running Galaxy Instance on http://localhost:8080. You can log to this instance using admin@galaxy.org and admin as a password
```

Comment:
How about: The galaxy admin email address and password and galaxy admin password are controlled by the variables
`galaxy_admin:` and `galaxy_admin:_pw`. These can be changed in in the group or host variables file, and default to admin@galaxy.org and admin.